EFCore.Sqlite.NodaTime.Tests/LocalDateTimeQueryTests.Methods.PlusWeeks.verified.txt

```diff
@@ -1,8 +1,9 @@
 ﻿{
-  target: 2020-10-24T23:42:16.321,
+  target: DateTimeOffset_1,
```

Comment:
I don't think these should be scrubbed. They're kinda the point of the test 😅

Reply:
try now